### PR TITLE
Add check feature and local storage

### PR DESCRIPTION
**Pull request by danalvrz**
- Added statusCheck.js and import from there feature for checking completed status on items
- Added local storage feature so the user gets the correct list values after page reloads

### CODE REVIEW

File: src/collection.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable no-unused-vars */
+let taskCollection = [];
```
Comment (mikethreels) [inline src/collection.js:2]: - [x] good job on setting up the project 👍 however the link to the live demo in the readme file is not working please fix it or remove the link from your readme file.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Dante, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/collection.js (hunk 1 above)
Comment (danalvrz) [reply src/collection.js:2]: Thanks for the review, Mike!

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @danalvrz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**